src/main/java/org/deepsymmetry/beatlink/CdjStatus.java

```diff
@@ -645,10 +644,10 @@ public CdjStatus(DatagramPacket packet) {
         if (Util.isOpusQuad(deviceName)) {
             trackSourcePlayer = translateOpusPlayerNumbers(packetBytes[40]);
             trackSourceSlot = findOpusTrackSourceSlot();
-            // Indicate whether we have a metadata archive available for the SD slot:
-            packetBytes[115] = (byte) ((OpusProvider.getInstance().findDatabase(SlotReference.getSlotReference(1, TrackSourceSlot.SD_SLOT)) == null) ? 4 : 0);
-            // Indicate whether we have a metadata archive available for the USB slot:
-            packetBytes[111] = (byte) ((OpusProvider.getInstance().findDatabase(SlotReference.getSlotReference(1, TrackSourceSlot.USB_SLOT)) == null) ? 4 : 0);
+            // Indicate that we want to listen for archives on the SD slot:
```

Comment:
Hmm, it seems we won’t need to be doing this because we are always going to be using the USB slot and different device numbers?

Reply:
Also, this is wrong because we should only be reporting this when there is a metadata archive mounted for the device/slot combination.

Reply:
I removed this trying to fix something thinking this was the cause of the problem. I just added it back and tested and all is working. I have rolled back this change. Thanks, good catch!